### ISSUE TITLE
Error on ha start with 0.3.0-beta

### ISSUE THREAD

**Issue by Krocko** [daenny/climate_group#16]
With 0.3.0-beta I get the following error after ha start:
```
Logger: homeassistant.components.climate
Source: custom_components/climate_group/climate.py:252 
Integration: Klima (documentation, issues) 
First occurred: 14:16:58 (1 occurrences) 
Last logged: 14:16:58

Error while setting up climate_group platform for climate
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 186, in _async_setup_platform
    await asyncio.gather(*pending)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 295, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 445, in _async_add_entity
    await entity.async_added_to_hass()
  File "/config/custom_components/climate_group/climate.py", line 114, in async_added_to_hass
    await self.async_update()
  File "/config/custom_components/climate_group/climate.py", line 252, in async_update
    self._preset = Counter(itertools.chain(all_presets)).most_common(1)[0][0]
IndexError: list index out of range
```

**Comment by daenny** [comment]
Ah yes. That could be if there is no state is in the list yet. I will fix it probably tonight. Thanks for reporting the issue.

**Comment by daenny** [comment]
Should be fixed in beta 0.3.1

**Comment by Krocko** [comment]
That was fast! Thank you very much, problem solved.
